MA-Bylaws.md

```diff
@@ -549,8 +452,7 @@ MAPs are encouraged to apply for approval of their results by MA as MA
 standard according to § 2.
 
 MAPs must make the results of the project work publicly available on the
-web under an open source license (usually the Modelica License or an
-open source license listed at <http://www.opensource.org/licenses>).
+web under an open source license that is listed at <http://www.opensource.org/licenses>.
```

Comment:
```suggestion
web under an open source license approved by the Open Source Initiative, see <http://www.opensource.org/licenses>.
```

Reply:
Decision at the MA board meeting on Jan. 20:

*MAPs must make the results of the project work publicly available on the web under an open source license (usually an open source license approved by the Open Source Initiative, see <http://www.opensource.org/licenses>).*

Reply:
added in https://github.com/modelica/MA-Bylaws/pull/5/commits/5408f0bc3e2c24687bef79c21f1f28ea7bf7f7f3